.github/workflows/rust-publish-dry-run-v2.yml

```diff
@@ -30,16 +30,25 @@ jobs:
     env:
       CARGO_BUILD_TARGET: ${{ inputs.target }}
       CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
+      RUST_BACKTRACE: 1
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
 
     - uses: stellar/actions/rust-cache@main
 
     - run: rustup update
     - run: rustup target add ${{ inputs.target }}
-    - if: inputs.target == 'aarch64-unknown-linux-gnu'
+
+    - name: Install add'l compilers (Linux arm64 only)
+      if: inputs.target == 'aarch64-unknown-linux-gnu'
       run: sudo apt-get update && sudo apt-get -y install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
 
+    - name: Update GNU Make (macOS only)
+      if: inputs.target == 'x86_64-apple-darwin' || inputs.target == 'aarch64-apple-darwin'
```

Comment:
I would add a comment indicating why this is needed. It is a but too specific to be self-explanatory

Reply:
Also, you can use `[endsWith](https://docs.github.com/en/actions/learn-github-actions/expressions#endswith)(inputs.target, "-apple-darwin")`

Reply:
done